# Takes 2 points and returns a status and a vector of points
geometry_msgs/Point posStart
geometry_msgs/Point posEnd
---
bool success
geometry_msgs/Point[] path
